pacoloco.tmpfiles.d

```diff
@@ -1 +1 @@
-d /var/cache/pacoloco - pacoloco pacoloco
+d /var/cache/pacoloco - pacoloco pacoloco
```

Comment:
it looks like the newline is removed. What is the reason of doing it?

Reply:
It's a mistake, nice catch!